docs/how-to/amplication-ai-examples.md

```diff
@@ -28,59 +28,105 @@ Can you create a new service for a blog application that will have posts, catego
 
 ![](./assets/amplication-ai/new-service.png)
 
-## Creating an Entity with Fields
+## Create New Entities
 
-Jovu can assist you in creating new entities and specifying the required fields, making it easier to define your service's data models.
+Jovu can assist you in [creating new entities](/set-up-entities/) and specifying the required fields, making it easier to define your service's data models.
 
 **Question**
 ```
-Can you help me create a new "Customer" entity with fields for name, email, and phone number?
+I want to add product reviews to my E-Commerce service. Can you set up the structure for customer reviews, review ratings, and a way to link reviews to specific products?
 ```
 
-**Jovu's Response**
+![Create New Entities with Jovu, Amplication AI](./assets/amplication-ai/create-entities.png)
+
+## Add Fields to Existing Entities
+
+Jovu can help you extend your existing entities by adding new fields to capture additional data exactly how you need it.
+
+**Question**:
 ```
-The "Customer" entity has been successfully created with fields for name, email, and phone number. You can view and customize the entity through this Customer Entity Link. Additionally, the APIs for Customer and related data are accessible here.
+For the authors in my BlogService, can you add a short bio section, a spot for a profile picture, and a place to include links to their social media profiles?
+```
+
+![Add Fields To Existing Amplication Entities](./assets/amplication-ai/create-entity-fields.png)
 
-Is there anything else you would like to add or modify for your blog application?
+## Create New Modules
+
+Jovu can assist you in creating [new custom modules](/custom-types-and-actions#modules) for your service to help you better organize your service and encapsulate related functionality.
+
+**Question**
 ```
+In the Checkout module, I need to define the data structure for the information required when a user initiates a payment. Could you help me create a DTO that includes the payment amount, the currency, and the selected payment method?
```

Comment:
@dericksozo - The question is not suitable to the answer and to the "create new modules" title